app/assets/javascripts/discourse/app/components/group-flair-inputs.js

```diff
@@ -15,23 +16,26 @@ export default Component.extend({
     return Discourse.getURL("/images/avatar.png");
   },
 
-  @discourseComputed("model.flair_url")
-  flairPreviewIcon(flairURL) {
-    return flairURL && /fa(r|b?)-/.test(flairURL);
+  @discourseComputed("model.flair_type")
+  flairPreviewIcon(flairType) {
+    return flairType && flairType === "icon";
   },
 
-  @discourseComputed("model.flair_url", "flairPreviewIcon")
-  flairPreviewIconUrl(flairURL, flairPreviewIcon) {
-    return flairPreviewIcon ? convertIconClass(flairURL) : "";
+  @discourseComputed("model.flair_icon")
+  flairPreviewIconUrl(flairIcon) {
+    return flairIcon ? convertIconClass(flairIcon) : "";
   },
 
-  @observes("model.flair_url")
-  _loadSVGIcon() {
-    debounce(this, this._loadIcon, 1000);
+  @on("didInsertElement")
+  @observes("model.flair_icon")
```

Comment:
maybe you can try to replace all of this by @on("didReceiveAttrs")

not sure it would work didn't pull the pr